js/ace.js

```diff
@@ -0,0 +1,408 @@
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'limits': limits,
+                'precision': {
+                    'price': undefined,
+                    'amount': undefined,
+                },
+                'active': undefined,
+                'info': id,
+            });
+        }
+        return result;
+    }
+
+    parseTicker (ticker, market = undefined) {
+        //
+        //     {
+        //         "base_volume":229196.34035399999,
+        //         "last_price":11881.06,
+        //         "quote_volume":19.2909
+        //     }
+        //
+        const marketId = this.safeString (ticker, 'id');
+        market = this.safeMarket (marketId);
+        const symbol = this.safeString (market, 'symbol');
+        return this.safeTicker ({
+            'symbol': symbol,
+            'timestamp': undefined,
+            'datetime': undefined,
+            'high': undefined,
+            'low': undefined,
+            'bid': undefined,
+            'bidVolume': undefined,
+            'ask': undefined,
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': undefined,
+            'close': this.safeString (ticker, 'last_price'),
+            'last': this.safeString (ticker, 'last_price'),
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': undefined,
+            'average': undefined,
+            'baseVolume': this.safeString (ticker, 'base_volume'),
+            'quoteVolume': this.safeString (ticker, 'quote_volume'),
+            'info': ticker,
+        }, market);
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        /**
+         * @method
+         * @name ace#fetchTicker
+         * @description fetches a price ticker, a statistical calculation with the information calculated over the past 24 hours for a specific market
+         * @param {string} symbol unified symbol of the market to fetch the ticker for
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const response = await this.publicGetListTradePrice (params);
+        const marketId = market['id'];
+        const ticker = this.safeValue (response, marketId, {});
+        //
+        //     {
+        //         "BTC/USDT":{
+        //             "base_volume":229196.34035399999,
+        //             "last_price":11881.06,
+        //             "quote_volume":19.2909
+        //         }
+        //     }
+        //
```

Comment:
instead of having the sub-response here, include the raw response from `publicGetListTradePrice` below the line `const response = await this.publicGetListTradePrice (params);`